### PR TITLE
Add Fanboy Newsletter list

### PR DESCRIPTION
**Pull request by ryanbr**
Adds Fanboy's Anti-Newsletter List. Address: https://github.com/brave/brave-browser/issues/28115

Both PEM's uploaded too 1pass:

`ad-block-updater-690FF3B4-8B6B-4709-8505-fEC6643D7BD9.pem`
`ad-block-updater-key2-690FF3B4-8B6B-4709-8505-fEC6643D7BD9.pem`

### CODE REVIEW

**Review by antonok-edm — APPROVED** [review]

**Review by antonok-edm — CHANGES_REQUESTED** [review]
Can you rename the second PEM?

It should be

`ad-block-updater-kdddfellohomdnfkdhombbddhojklibj.pem`

Also the first one should use the updated uppercase UUID

File: filter_lists/regional.json [hunk 1]
```diff
@@ -179,6 +179,20 @@
             "base64_public_key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAs6ib1E68QTHuz/TdEn+ck++Okbcf5xDfQjkhpTb59fnjm0rjII6MV3rWLEj5dkOvpAVzDzFCcyC/JJ/SYIY7nFRQl+ptttd1nLWNu7qewFGh2tezwuEMzPlzQr+KK93Yh0dooCC9KniR/0+kpYeGqBC8tBGz8f4KoV+/zTfkD/47pztflCxNJfrAj4P0GDQhvz8lBtnm6Pa7tGOMhksnMsYWRWsYs6uvjsfd+3uYbiCYa7lEMjZZKy7dHg8hPSUeGZo28kTqYFXufH5jY3q7Luk+dQey1+m1X97KLPOtmwaLFCFqWBha5qFMg/mgzczlaphReFUycwrYVjjOICKrYwIDAQAB"
         }
     },
+    {
+        "uuid": "690FF3B4-8B6B-4709-8505-fEC6643D7BD9",
```
Comment (antonok-edm) [inline filter_lists/regional.json:183]: UUID shouldn't contain any lowercase letters

**Comment by ryanbr** [comment]
Renamed pems also

**Comment by ryanbr** [comment]
Looks good @antonok-edm ?

**Review by antonok-edm — APPROVED** [review]

**Pull request merged**